src/exposurelog/routers/add_message.py

```diff
@@ -29,9 +30,14 @@ async def add_message(
     ),
     instrument: str = fastapi.Body(
         default=...,
-        description="Short name of instrument (e.g. HSC)",
+        description="Short name of instrument (e.g. LSSTCam)",
     ),
     message_text: str = fastapi.Body(..., description="Message text"),
+    tags: typing.List[str] = fastapi.Body(
+        default=[],
+        description="Tags describing the message, as space-separated words. "
```

Comment:
Ah, so tags are always associated with a message and are not independently set.

Reply:
The list of allowed tags will probably be set by the front end. But the service performs no validation beyond making sure they meet our naming requirement. I had originally designed narrativelog to take a list of allowed tags as configuration, but then we realized we didn't really know which tags we wanted to use. So at least for now free for all. I hope at some point to add an API call to return the list of tags presently in use.